cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -0,0 +1,380 @@
+        """
+        Remove certificate from the course
+        """
+        for index, cert in enumerate(course.certificates['certificates']):
+            if int(cert['id']) == int(certificate_id):
+                course.certificates['certificates'].pop(index)
+                store.update_item(course, request.user.id)
+                break
+        return JsonResponse(status=204)
+
+    # pylint-disable: unused-variable
+    @staticmethod
+    def remove_signatory(request, store, course, certificate_id, signatory_id):
+        """
+        Remove the specified signatory from the provided course certificate
+        """
+        for cert_index, cert in enumerate(course.certificates['certificates']):  # pylint: disable=unused-variable
+            if int(cert['id']) == int(certificate_id):
+                for sig_index, signatory in enumerate(cert.get('signatories')):  # pylint: disable=unused-variable
+                    if int(signatory_id) == int(signatory['id']):
+                        del cert['signatories'][sig_index]
+                        store.update_item(course, request.user.id)
+                        break
+        return JsonResponse(status=204)
+
+
+class Certificate(object):
+    """
+    The logical representation of an individual course certificate
+    """
+    def __init__(self, course, certificate_data):
+        """
+        Instantiate a Certificate object instance using the provided information.
+        """
+        self.course = course
+        self.certificate_data = certificate_data
+        self.id = certificate_data['id']  # pylint: disable=invalid-name
+
+    def get_certificate_data(self):
+        """
+        Retrieve the locally-stored certificate data from the Certificate object via a helper method
+        """
+        return self.certificate_data
+
+
+@login_required
+@require_http_methods(("GET", "POST"))
+@ensure_csrf_cookie
+def certificates_list_handler(request, course_key_string):
+    """
+    A RESTful handler for Course Certificates
+
+    GET
+        html: return Certificates list page (Backbone application)
+    POST
+        json: create new Certificate
+    """
+    course_key = CourseKey.from_string(course_key_string)
+    store = modulestore()
+    with store.bulk_operations(course_key):
+        try:
+            course = _get_course_and_check_access(course_key, request.user)
+        except PermissionDenied:
+            msg = _('PermissionDenied: Failed in authenticating {user}').format(user=request.user)
+            return JsonResponse({"error": msg}, status=403)
+
+        if 'text/html' in request.META.get('HTTP_ACCEPT', 'text/html'):
+            certificate_url = reverse_course_url('certificates.certificates_list_handler', course_key)
+            course_outline_url = reverse_course_url('course_handler', course_key)
+            certificates = None
+            print 'VIEW SETTING: {}'.format(settings.FEATURES.get('CERTIFICATES_HTML_VIEW'))
```

Comment:
Not sure that this (and below) `print` statement is particularly useful - maybe `log.debug` instead?